app/Policies/RatingPolicy.php

```diff
@@ -91,4 +91,9 @@ public function forceDelete(User $user, Rating $rating)
   {
     //
   }
+
+  protected function hasRated(User $user)
+  {
+    return boolval(Rating::where(['user_id'=>$user->id])->get()->all());
```

Comment:
Would `exists()` be more appropriate here?

https://laravel-recipes.com/laravel-checking-if-a-record-exists/